telegraf/manifests/deis-monitor-telegraf-daemon.yaml

```diff
@@ -41,6 +41,13 @@ spec:
             value: "metrics"
           - name: "NSQ_ENDPOINTS"
             value: http://$(DEIS_NSQD_SERVICE_HOST):$(DEIS_NSQD_SERVICE_PORT_HTTP)
+          - name: DEIS_LOGGER_REDIS_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: logger-redis-creds
+                key: password
+          - name: "REDIS_SERVERS"
+            value: tcp://:$(DEIS_LOGGER_REDIS_PASSWORD)@$(DEIS_LOGGER_REDIS_SERVICE_HOST):$(DEIS_LOGGER_REDIS_SERVICE_PORT)
```

Comment:
Although the password gets put into an env var, note that the source is from a secret. The only other alternative is mounting that secret on the file system, but that isn't any more secure than this is.